spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/ReactorLoadBalancerExchangeFilterFunction.java

```diff
@@ -61,10 +62,13 @@
 
 	private final LoadBalancerProperties properties;
 
+	private final List<LoadBalancerClientRequestTransformer> transformers;
+
 	public ReactorLoadBalancerExchangeFilterFunction(ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory,
-			LoadBalancerProperties properties) {
+			LoadBalancerProperties properties, List<LoadBalancerClientRequestTransformer> transformers) {
```

Comment:
We cannot modify constructors after the major GA release. We can only add an additional constructor.

Reply:
Fixed in https://github.com/spring-cloud/spring-cloud-commons/pull/874/commits/e1c5d5aaeef7ce361befbb78aae20be7070d12eb

Reply:
Thanks, let's also deprecate the old one.

Reply:
Deprecated in b23f05fe5afc40a7dfa31fdccb39ffc819872079